src/core/main/ts/fmt/ApplyFormat.ts

```diff
@@ -156,7 +157,7 @@ const applyFormat = function (ed, name, vars?, node?) {
 
         // Can we rename the block
         // TODO: Break this if up, too complex
-        if (contentEditable && !hasContentEditableState && format.block &&
+        if (/*contentEditable && !hasContentEditableState &&*/  format.block &&
```

Comment:
We need to use the setting you created here as well, instead of commenting code out.

Reply:
I've commented these 2 conditions because we don't need them. If I were to use new makeNonEditableStylable i have to duplicate and wrap the above function.